megamek/src/megamek/common/Building.java

```diff
@@ -51,16 +59,19 @@
     private Vector<Coords> coordinates = new Vector<Coords>();
 
     /**
-     * The construction type of the building.
+     * The Construction Type of the building; equal to the terrain elevation of the BUILDING terrain of a hex.
+     * Types: LIGHT, MEDIUM, HEAVY, HARDENED, WALL
      */
     private int type = Building.UNKNOWN;
 
     /**
      * The Basement type of the building.
      */
     private Map<Coords,BasementType> basement = new HashMap<Coords,BasementType>();
+    
     /**
-     * the class of the building
+     * The Building Class of the building; equal to the terrain elevation of the BUILDING CLASS terrain of a hex.
+     * Types: STANDARD, HANGAR, FORTRESS, GUN_EMPLACEMENT 
      */
     private int bldgClass = Building.STANDARD;
```

Comment:
Maybe move this to be listed under the classes instead of having a comment that might go out of date?